.ci/.bump-stack-version.yml

```diff
@@ -0,0 +1,55 @@
+---
+## This is the file that contains what projects should bump their elastic stack
+## versions.
+##
+## The data structure:
+##  repo: the repository name
+##  script: the relative path to the script in charge to bump the version
+##  branches: the list of branches that should benefit from this automation.
+##  enabled: whether the automation has been enabled for this project.
+##  reusePullRequest: whether to reuse the existing opened Pull Request.
+##  labels: list of GitHub labels to be added to the Pull Request (comma separated).
+##
+##  NOTE:
+##  <minor> is a special token that refers to the latest minor branch. The automation
+##          will resolve the correct version at runtime, so you don't need to change it.
+##
+projects:
+  - repo: apm-server
+    script: .ci/bump-stack-version.sh
+    branches:
+      - master
+      - 8.<minor>
+      - 7.<minor>
+    enabled: true
+    reusePullRequest: false
+    labels: dependency,backport-skip
+  - repo: beats
+    script: .ci/bump-stack-version.sh
+    branches:
+      - master
+      - 8.<minor>
+      - 7.<minor>
+      - "6.8"
+    enabled: true
+    reusePullRequest: false
+    labels: dependency,build-monitoring,backport-skip,Team:Beats-On-Call
+  - repo: e2e-testing
+    script: .ci/bump-stack-version.sh
+    branches:
+      - main
+      - 8.<minor>
+      - 7.<minor>
```

Comment:
do you want to use minor-1?

Reply:
Yes, let me change it, and I need another PR removing the file from oblt-dev

Reply:
Since it uses `main` -> https://github.com/elastic/apm-pipeline-library/pull/1503/files#diff-eba3c46469d30c72b2ce74f5be13c5b7d85e69e0d8ef95dba6e5bf8e3f09a9e8R20 as long it gets merged then it should work :)